README.md

```diff
@@ -216,13 +216,37 @@ Currently, the token, when decoded, should contain:
 
 ## Demo
 
```

Comment:
let's had a subsection

###requirements

where we list required services to run the demo:
- docker
- jq (if you apply my suggestion below, otherwise, python)